lexos/managers/utility.py

```diff
@@ -1462,34 +1405,27 @@ def generate_z_test_top_word(file_manager: FileManager):
 
 
 def get_top_word_csv(test_results, csv_header):
```

Comment:
if you change the data structure in `generate_topwords` to data frame, you will be able to directly save them as csv. 
http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_csv.html